feature-libs/checkout/scheduled-replenishment/components/checkout-schedule-replenishment-order/checkout-schedule-replenishment-order.component.html

```diff
@@ -105,11 +164,36 @@
         </select>
       </div>
       <div *ngIf="isMonthly" class="cx-dayMonth">
-        <span class="form-data-label">{{
-          'checkoutScheduledReplenishment.dayOfMonth' | cxTranslate
-        }}</span>
+        <span
+          class="form-data-label"
+          *cxFeature="'!a11yScheduleReplenishment'"
+          >{{ 'checkoutScheduledReplenishment.dayOfMonth' | cxTranslate }}</span
+        >
+        <label
+          class="form-data-label"
+          *cxFeature="'a11yScheduleReplenishment'"
+          for="order-replenishment-day-of-month"
+          >{{
+            'checkoutScheduledReplenishment.dayOfMonth' | cxTranslate
+          }}</label
+        >
         <div class="cx-day-of-month">
           <select
+            *cxFeature="'!a11yScheduleReplenishment'"
+            class="form-control"
+            (change)="changeDayOfTheMonth($event.target.value)"
+          >
+            <option
+              *ngFor="let nDays of numberOfDays"
+              [value]="nDays"
+              [selected]="nDays === scheduleReplenishmentFormData.nthDayOfMonth"
+            >
+              {{ nDays }}
+            </option>
+          </select>
+          <select
+            *cxFeature="'a11yScheduleReplenishment'"
+            id="order-replenishment-day-of-month"
```

Comment:
We can introduce `id` attributes without a feature toggle because its not a breaking change.

Reply:
I've just updated it.